spec/test.js

```diff
@@ -0,0 +1,40 @@
+const { TestScheduler } = require('jest');
+const puppeteer = require('puppeteer');
+const pageUrl = 'http://localhost:3101/';
+
+let page, browser;
+const width = 1200;
+const height = 700;
```

Comment:
I'd swap down line 5 to be below these other lines, so that you can keep your `const`s together & similarly with your `let`s.